spec/models/user_spec.rb

```diff
@@ -946,9 +946,9 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
     it "returns the number of visits per day" do
       user.sign_in_count = 8
       user.created_at = Time.zone.today.days_ago(4)
-      expect(user.number_visits_per_day).to eq(2.0)
+      expect(user.number_visits_per_day).to eq(2)
       user.created_at = Time.zone.today
-      expect(user.number_visits_per_day).to eq(8.0)
+      expect(user.number_visits_per_day).to eq(8)
```

Comment:
removed the decimal place for readability and because it does not contribute useful information